hack/jenkins/e2e.sh

```diff
@@ -221,462 +122,336 @@ case ${JOB_NAME} in
 
   # Runs a subset of tests on GCE in parallel. Run against all pending PRs.
   kubernetes-pull-build-test-e2e-gce)
-    : ${E2E_CLUSTER_NAME:="jnks-e2e-gce-${NODE_NAME}-${EXECUTOR_NUMBER}"}
-    : ${E2E_NETWORK:="e2e-gce-${NODE_NAME}-${EXECUTOR_NUMBER}"}
-    : ${GINKGO_PARALLEL:="y"}
+    # XXX Not a unique project
+    export E2E_NAME="e2e-gce-${NODE_NAME}-${EXECUTOR_NUMBER}"
+    export GINKGO_PARALLEL="y"
     # This list should match the list in kubernetes-e2e-gce.
-    : ${GINKGO_TEST_ARGS:="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"}
-    : ${KUBE_GCE_INSTANCE_PREFIX:="e2e-gce-${NODE_NAME}-${EXECUTOR_NUMBER}"}
-    : ${PROJECT:="kubernetes-jenkins-pull"}
+    export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+    export FAIL_ON_GCP_RESOURCE_LEAK="false"
+    export PROJECT="kubernetes-jenkins-pull"
     # Override GCE defaults
-    NUM_NODES=${NUM_NODES_PARALLEL}
+    export NUM_NODES="6"
     ;;
 
   # GCE core jobs
 
   # Runs all non-slow, non-serial, non-flaky, tests on GCE in parallel.
   kubernetes-e2e-gce)
-    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e"}
-    : ${E2E_PUBLISH_GREEN_VERSION:="true"}
-    : ${E2E_NETWORK:="e2e-gce"}
+    # This is the *only* job that should publish the last green version.
+    export E2E_PUBLISH_GREEN_VERSION="true"
     # This list should match the list in kubernetes-pull-build-test-e2e-gce.
-    : ${GINKGO_TEST_ARGS:="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"}
-    : ${GINKGO_PARALLEL:="y"}
-    : ${KUBE_GCE_INSTANCE_PREFIX="e2e-gce"}
-    : ${PROJECT:="k8s-jkns-e2e-gce"}
-    : ${FAIL_ON_GCP_RESOURCE_LEAK:="true"}
+    export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+    export GINKGO_PARALLEL="y"
+    export PROJECT="k8s-jkns-e2e-gce"
     ;;
 
   # Runs slow tests on GCE, sequentially.
   kubernetes-e2e-gce-slow)
-    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e-slow"}
-    : ${E2E_NETWORK:="e2e-slow"}
-    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=\[Slow\] \
-                           --ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"}
-    : ${GINKGO_PARALLEL:="y"}
-    : ${KUBE_GCE_INSTANCE_PREFIX:="e2e-slow"}
-    : ${PROJECT:="k8s-jkns-e2e-gce-slow"}
-    : ${FAIL_ON_GCP_RESOURCE_LEAK:="true"}
+    export GINKGO_TEST_ARGS="--ginkgo.focus=\[Slow\] \
+                             --ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+    export GINKGO_PARALLEL="y"
+    export PROJECT="k8s-jkns-e2e-gce-slow"
     ;;
   
   # Runs all non-flaky, non-slow tests on GCE, sequentially,
   # and in a multi-zone ("Ubernetes Lite") cluster.
   kubernetes-e2e-gce-ubernetes-lite)
-    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e-ubelite"}
-    : ${E2E_PUBLISH_GREEN_VERSION:="true"}
-    : ${E2E_NETWORK:="e2e-gce-ubelite"}
-    : ${GINKGO_TEST_ARGS:="--ginkgo.skip=$(join_regex_allow_empty \
```

Comment:
Yes; these regex's don't exist anymore, and `GINKGO_TEST_ARGS` now has a sane default, so I figured I'd just leave it out, (similar to `flannel`, `upgrades`, etc.).